apps/dashboard/src/lib/components/Org/LandingPage/index.svelte

```diff
@@ -561,13 +561,7 @@
             {/if}
           </div>
         </ul>
-        <!-- <p class="text-center mt-5">
-          Powered by
-          <a class="text-primary-600 underline" href="https://classroomio.com" target="_blank"
-            >ClassroomIO</a
-          >
-        </p> -->
-        <div class="flex items-center justify-center">
+        <div class="sticky bottom-2 left-6 ml-auto">
```

Comment:
but why is the one on the right and the other one on the left.

Okay, just add the positioning within the component itself so it has the same positioning everywhere it is imported